todo.js

```diff
@@ -1 +1,173 @@
-// Write your code
+"use strict";
+
+function Root() {
+  const [contents, setContents] = React.useState(() => {
+    let obj = {};
+    const ids = Object.keys(localStorage)
+      .filter((id) => id !== "id")
+      .map((id) => {
+        return parseInt(id);
+      })
+      .sort((a, b) => a - b);
+
+    ids.map((id) => {
+      obj[id] = localStorage.getItem(id);
+    });
```

Comment:
こういうのはmap（写像）ではないので、代わりに`forEach`を使った方が意図が明確になってベターです。
```suggestion
    ids.forEach((id) => {
      obj[id] = localStorage.getItem(id);
    });
```